viomem/sys/viomem.c

```diff
@@ -627,6 +627,66 @@ LONGLONG FindConsecutivePagesCountMDL(PPFN_NUMBER Pages, LONGLONG Remaining)
 	return index;
 }
 
+//
+// Function calculates and returns count of memory ranges from MDL.
+//
+
+ULONG GetMemoryRangesCountFromMdl(PMDL Mdl)
+{
+	PPFN_NUMBER pfnNumbers = 0;
```

Comment:
I think this name is very confusing. It looks like it is going to hold the amount of PFNs and not the pointer to the array.